src/Microsoft.Health.Dicom.Core/Modules/ServiceModule.cs

```diff
@@ -110,7 +111,7 @@ public void Load(IServiceCollection services)
                 .AsImplementedInterfaces();
 
             services.Add<DicomTagParser>()
-                         .Scoped()
+                         .Singleton()
                          .AsSelf()
                          .AsImplementedInterfaces();
```

Comment:
I am unfamiliar with these methods -- I'm looking at them now, but based on their names and usage I am assuming they are probably equivalent to the following (as long as the class only implements 1 interface and we don't access `DicomTagParser` through the concrete type).

```csharp
services.AddSingleton<IDicomTagParser, DicomTagParser>();
```